Core/Core/Router/Router.swift

```diff
@@ -191,4 +191,32 @@ public class Router: RouterProtocol {
         }
         fallback(url, from, options)
     }
+
+    public static func open(url: URLComponents) {
+        var components = url
+        // Canonicalize relative & schemes we know about.
+        switch components.scheme {
+        case "http", "canvas-courses", "canvas-student", "canvas-teacher", "canvas-parent":
+            components.scheme = "https"
```

Comment:
Would this make it so outbound links to http-only pages fail? May also cause issues with minicanvas, so I'm triggering a nightly to make sure.

Reply:
Oh, I assumed we were always forcing https, but looking again, that's not what we used to do. I can revise this logic to preserve rather than upgrade `http`.